types/oauth2-server/index.d.ts

```diff
@@ -183,6 +237,11 @@ declare namespace OAuth2Server {
          * Always revoke the used refresh token and issue a new one for the refresh_token grant.
          */
         alwaysIssueNewRefreshToken?: boolean;
+
+        /**
+         * Additional supported grant types.
+         */
+        extendedGrantTypes?: {[key: string]: typeof AbstractGrantType };
```

Comment:
I think generally object types have spaces inside the braces in this repo? Either way, the brace spacings are inconsistent.